src/components/ModalContent/ModalContent.js

```diff
@@ -98,6 +98,10 @@ class ModalContent extends React.Component {
     okapi: PropTypes.object
   };
 
+  static defaultProps = {
+    toggleButton: () => {},
```

Comment:
You can use the `noop` function from `lodash` which does the same.
```suggestion
    toggleButton: noop,
```

Reply:
fixed